runtime/jvmMain/src/kotlinx/benchmark/jvm/JvmBenchmarkRunner.kt

```diff
@@ -56,6 +56,21 @@ fun main(args: Array<String>) {
         }
     }
 
+    val profilerName = config.advanced["jvmProfiler"]
+    when (profilerName) {
+        "gc" -> jmhOptions.addProfiler("gc")
+        "stack" -> jmhOptions.addProfiler("stack")
+        "cl" -> jmhOptions.addProfiler("cl")
+        "comp" -> jmhOptions.addProfiler("comp")
+        "perf" -> jmhOptions.addProfiler("perf")
+        "perfnorm" -> jmhOptions.addProfiler("perfnorm")
+        "perfasm" -> jmhOptions.addProfiler("perfasm")
+        "xperfasm" -> jmhOptions.addProfiler("xperfasm")
+        "dtraceasm" -> jmhOptions.addProfiler("dtraceasm")
+        null -> {}
+        else -> throw IllegalArgumentException("Invalid value for 'jvmProfiler': $profilerName. Accepted values: gc, stack, cl, comp")
+    }
```

Comment:
Voting for `jmhOptions.addProfiler(profilerName)`:
- the list of profilers here is already incomplete;
- we can't hardcode it as users are allowed to specify a custom JMH version and it may have different profilers bundled inside;
- there's no way to specify a custom profiler bundled in a separate jar.